packages/shopify-codemod/transforms/global-assignment-to-default-export.js

```diff
@@ -12,7 +12,7 @@ export default function globalAssignmentToDefaultExport({source}, {jscodeshift:
   }
 
   function isDefaultExportMemberAssignment(defaultExportName, assignmentTargetString) {
-    return new RegExp(`^${defaultExportName}\..+`).test(assignmentTargetString);
+    return new RegExp(`^${defaultExportName}\\..+`).test(assignmentTargetString);
```

Comment:
The new eslint caught this 👌 
Need to escape the escape in order for the regex to match the literal period.